plugin/evm/vm.go

```diff
@@ -550,13 +550,9 @@ func (vm *VM) Initialize(
 }
 
 func (vm *VM) initializeMetrics() error {
+	metrics.Enabled = true
 	vm.sdkMetrics = prometheus.NewRegistry()
-	// If metrics are enabled, register the default metrics registry
-	if !metrics.Enabled {
-		return nil
-	}
```

Comment:
isn't this something we want to keep?

Reply:
From the previous logic, we always have metrics enabled, see

https://github.com/ava-labs/subnet-evm/blob/b049381bf3f50eb8687ca131c5237abdc0d2aa29/metrics/metrics.go#L13

If at some point we want a configuration option to disable metrics, we can refactor this code, but for now there is not even an option to disable/enable metrics.